OloEngine/src/OloEngine/Serialization/MeshBinarySerializer.cpp

```
@@ -0,0 +1,1225 @@
+                OMeshFormat::BoneInfluenceHeader biHeader;
+                ReadBytes(payload, &biHeader, sizeof(biHeader));
+
+                // Validate bone influence stride matches current struct.
+                if (biHeader.InfluenceStride != sizeof(BoneInfluence))
+                {
+                    OLO_CORE_ERROR("MeshBinarySerializer::Read: BoneInfluence stride mismatch (file {}, expected {}) in '{}'",
+                                   biHeader.InfluenceStride, sizeof(BoneInfluence), path.string());
+                    return nullptr;
+                }
+
+                if (biHeader.InfluenceCount > 0 && biHeader.EncodedSize > 0)
```

Clang (14.0.6)

[note] +2, including nesting penalty of 1, nesting level increased to 2

(clang)